CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# Contributing
+
+## Reporting Issues
+
+If you have found what you think is a bug, please [file an issue](https://github.com/pmndrs/jotai/issues/new).
```

Comment:
I would probably add something about recommending a minimal reproduction of the bug on https://codesandbox.io/ along with an easily fork-able link with a fully setup codesandbox (this can be added at a later stage, but is a nice way to speed-up contribution).